website/static/js/home-page/newAndNoteworthyPlugin.js

```diff
@@ -0,0 +1,162 @@
+/**
+ * New and Noteworthy Projects
+ */
+
+var $ = require('jquery');
+var m = require('mithril');
+var $osf = require('js/osfHelpers');
+var Raven = require('raven-js');
+
+// CSS
+require('css/new-and-noteworthy-plugin.css');
+
+// XHR config for apiserver connection
+var xhrconfig = function(xhr) {
+    xhr.withCredentials = true;
+};
+
+
+var NewAndNoteworthy = {
+    controller: function() {
+        var self = this;
+        self.newAndNoteworthyNodes = m.prop([]); // New and noteworthy nodes list
+        self.popularNodes = m.prop([]); // Most popular nodes list
+        self.contributorsMapping = {}; // Dictionary mapping node id to some of the contrib names and the contrib total.
+        self.SHOW_TOTAL = 5; // Number of new and noteworthy projects displayed in each column
+        self.errorLoading = m.prop(false);  // True if error retrieving projects or contributors.
+        self.someDataLoaded = m.prop(false); // True when the initial request to retrieve projects is complete.
```

Comment:
This shouldn't need to be a prop, since it's derived data.

```js
self.someDataLoaded = function() { return Boolean(self.newAndNoteworthyNodes().length); };
```